GUIDELINES.adoc

```diff
@@ -0,0 +1,96 @@
+
+.AsciiDoc source for a new glossary entry
+----
+[discrete]
+[[<id>]]
+==== <term> (<class>)
+*Description*: <description>
+
+*Use it*: <yes/no/with caution>
+
+*Incorrect forms*: <incorrect-form>
+
+*See also*: xref:<another-term-id>[<another-term>]
+----
+
+[options="header"]
+|======================
+|Replace                |With                               | Notes
+|`<id>`                 |The term in lowercase.             |If the term consists of more than one word, separate them with a hyphen ("-"). Do not use spaces, capital letters, or special characters (for example, slash "/") in the ID.
+|`<term>`               |The term to be described.          |
+|`<description>`        |The description of the term.|Use full sentences with periods. If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. Add information about why we should use the term in this form. If we should not use the term, explain why. Alternatively, explain why the terms listed in the *Incorrect forms* field are incorrect.
+|`<yes/no/with caution>`|"yes" if the term should be used.
+
+"no" if the term should not be used.
+
+"with caution" if the term should be used sporadically or only in certain cases.|Explain why we should or should not use the term in the *Description* field.
+|`<class>`              |The part of speech of the term.|Possible values: noun, verb, adjective, adverb, pronoun, conjunction, preposition. Use noun for acronyms or abbreviations. Leave the field empty for symbols.
+|`<incorrect-form>`     |The form of the term that we should not use.|If there are multiple incorrect forms, separate them with a comma (",").
+|`<another-term-id>`    |The ID of another term related to this one.|Always verify that this ID already exists in the document; otherwise, it will fail to validate if the ID does not exist. Do not include a space between `<another-term-id>` and `[<another-term>]`.
+|`<another-term>`       |The term related to this one.|To link to another document or a web page, replace `xref:<another-term-id>[<another-term>]` with `<link>[<name-of-the-document/web-page>]`.
+|======================
+
+.Additional glossary entry guidelines
+* Do not combine two terms into one entry.
+
+* Keep all fields, even if a field is empty. An empty field serves as a placeholder if the field is required later.
+
+* Use quotation marks on the first occurrence of a term in the *Description* field. Do not include an acronym in parentheses within the quotation marks; for example, "Asynchronous Transfer Mode" (ATM) is a network technology based on transferring data in cells or packets of a fixed size.
+
+* If the *Description* field contains more than one paragraph, only the last paragraph is displayed in the Atom linter's text. Ensure that word usage advice is contained in the last paragraph.
+
+* Use single lines with no breaks for each paragraph. That is, in a text editor, it looks like this:
```

Comment:
The second sentence here reads a bit awkwardly. Maybe just "In a text editor, it looks like this:"

Reply:
Agree, I just got rid of that example, it seems unnecessary to try to give an example of how an editor might look (which depends on the editor). Instead I added "In other words, do not hard wrap at 80 characters, for example."

Reply:
👍